vivo-project/VIVO
# Remove Spring Framework dependency and directly instantiate visualiza…

…tion handlers.

**[VIVO-3686](https://github.com/vivo-project/VIVO/issues/3686)**:

# What does this pull request do?
Removes Spring Framework dependency and builds map of visualization request handlers directly without using Spring dependency injection XML.

# How should this be tested?
* Build VIVO
* Load [sample data](https://github.com/vivo-project/sample-data) if necessary.
* Confirm that all visualizations are working normally (capability map; individual-level sparkline, Map of Science and co-author/co-investigator chord diagram; organization-level temporal graph and Map of Science)

# Interested parties
@VIVO-project/vivo-committers
